### PR TITLE
[Behat] Add xpath Helper class

### PR DESCRIPTION
**Pull request by miguelcleverti**
Helper class containing generic methods for element finding using Xpath. Moved from BehatBundle

### CODE REVIEW

File: eZ/Bundle/PlatformBehatBundle/Helper/Xpath.php [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+     * @param string $search String to search
+     *
+     * @return string XPath for the element/object
+     */
+    public function makeElementXpath($element, $search)
+    {
+        $selectorsHandler = $this->session->getSelectorsHandler();
+        $literal = $selectorsHandler->xpathLiteral($search);
+
+        // To be able to work on mink 1.6 (ezplatform) & mink 1.5 (5.4+ezpublish-community) w/o deprecation exceptions
```
Comment (bdunogier) [inline eZ/Bundle/PlatformBehatBundle/Helper/Xpath.php]: This doesn't apply anymore, since this is ezplatform@master (6.4 and onward), does it ?

**Comment by bdunogier** [comment]
Is it really used that much ?

In all of the behat scenarii I have written lately, I've just used `assertSession()` calls, with manual CSS selectors. In which scenarii can I see it used ?

**Comment by dpobel** [comment]
I think we should use the CSS selectors instead of XPath expressions. In most cases, CSS selectors are enough and in any case easier to write/maintain/manage.

**Comment by bdunogier** [comment]
Do you think we could keep the Behat helper where it is (since it is used), and use CSS selectors from now on ? Or is it too much work, @miguelcleverti ?

**Comment by miguelcleverti** [comment]
@bdunogier I took better look at this helper and found out that it was only being used to find checkboxes input types.
I refactored the code that was using this helper ,so this is no longer needed.
With that said I will close this PR.

**Pull request closed without merging**